src/styles/styles.js

```diff
@@ -2860,6 +2860,21 @@ const styles = {
         fontSize: variables.fontSizeNormal,
         maxWidth: 240,
     },
+
+    welcomeTextHeader: {
+        fontSize: variables.fontSizeHero,
+    },
```

Comment:
perhaps this should just be `heroFont` or something and placed near the font styles so we could reuse this style?

Reply:
That works for me! Maybe something like `headerFont` since I think we'll put the new font in here as well once it exists